gtsam/linear/NoiseModel.h

```diff
@@ -897,6 +932,10 @@ namespace gtsam {
           double xc2 = (error*error)/csquared_;
           return std::exp(-xc2);
         }
+        double residual(double error) const {
```

Comment:
Is pretty horrible copy/paste. can we not just say `using Welsh = Welsch’ ??

Reply:
Yeah I would actually prefer that. This is jarring.